index.js

```diff
@@ -0,0 +1,29 @@
+// Utilizar funcionalidades del Ecmascript 6
+'use strict'
+
+// Cargamos el módulo de mongoose para poder conectarnos a MongoDB
+var mongoose = require('mongoose');
+
+// *Cargamos el fichero app.js con la configuración de Express
+var app = require('./app');
+
+// Creamos la variable PORT para indicar el puerto en el que va a funcionar el servidor
+var port = 3800;
```

Comment:
Esto deberías ponerlo en un fichero de configuración para poder cambiar el despliegue sin cambiar el código fuente o usar variables de entorno. Si lo hardcodeas no se puede modificar

Reply:
¿te refieres a algo de este estilo?:
https://codeburst.io/node-js-best-practices-smarter-ways-to-manage-config-files-and-variables-893eef56cbef